src/AsyncTCP.cpp

```diff
@@ -904,6 +1037,13 @@ int8_t AsyncClient::_poll(tcp_pcb* pcb){
         _close();
         return ERR_OK;
     }
+#if ASYNC_TCP_SSL_ENABLED
+    if(_pcb_secure && !_handshake_done && (now - _rx_last_packet) >= 5000){
```

Comment:
let's make this at least a define :) I think I also have left some hard-coded values here and there for timeouts

Reply:
#define it is....